ui/src/test/java/edu/wpi/grip/ui/codegeneration/tools/HelperTools.java

```diff
@@ -70,6 +73,17 @@ public static Mat bytedecoMatToCVMat(org.bytedeco.javacpp.opencv_core.Mat input)
 		return out;
 	}
 
+	public static void setEnumSocket(InputSocket sock, String id){
```

Comment:
If anybody has a better way to do this, it would be highly appreciated. The problem is all of the enums used are private to the class so we cannot actually access them. So the only way to access a specific one (without using an index in which case we don't know which one it is) that I could think of was comparing the toString method treating the enum as an object.